### ISSUE TITLE
jupyterlab_thredds

### ISSUE THREAD

**Issue by sverhoeven** [NLeSC/TEAM2018#13]
Create a paper of https://github.com/eWaterCycle/jupyterlab_thredds

It is a JupyterLab extension which has dataset browser for THREDDS catalog

Can inject iris/xarray/leaflet code cells into a Python notebook of a selected dataset to further process/visualize the dataset.

**Comment by jspaaks** [comment]
I may be able to help you compose some blurps for the RSD, which could serve as a start for (the abstract of) the paper.

**Comment by sverhoeven** [comment]
Just me.

**Comment by romulogoncalves** [comment]
@sverhoeven  during the Sprint please decide if you want to merge with #15 or co-author it.

**Comment by sverhoeven** [comment]
Work done in sprint:
* @nielsdrost wrote draft introduction
* tried out the extension with other peoples THREDDS servers and found numerous bugs and hard to ignore improvements
* Started with audience and ecosystem write up at https://github.com/eWaterCycle/jupyterlab_thredds/tree/blog
* Investigated how to speed up crawling of THREDDS catalog (https://github.com/eWaterCycle/jupyterlab_thredds/issues/23)
* Fixed some bugs and quality of life improvement implemented resulting in a new release

**Comment by romulogoncalves** [comment]
@sverhoeven were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by romulogoncalves** [comment]
@sverhoeven what is the current status?

**Comment by sverhoeven** [comment]
An abstract has been tried to be submitted to EGU 2019 conference, but failed due to bureaucratic reason.

**Comment by romulogoncalves** [comment]
For sure it can be re-submitted elsewhere with some re-edition. I consider the issue solved.
